app/containers/Admin/ReportingYear/NewReportingYearFormDialog.tsx

```diff
@@ -6,28 +6,9 @@ import {JSONSchema6} from 'json-schema';
 import FormObjectFieldTemplate from 'containers/Forms/FormObjectFieldTemplate';
 import FormFieldTemplate from 'containers/Forms/FormFieldTemplate';
 import newReportingYearSchema from './create_reporting_year.json';
-import AltDateInput from 'components/Forms/AltDateInput';
-import AltDateTimeInput from 'components/Forms/AltDateTimeInput';
+import DatePickerWidget from 'components/Forms/DatePickerWidget';
 import {validateAllDates, validateUniqueKey} from './reportingYearValidation';
-import {nowMoment} from 'functions/formatDates';
-
-function transformUiSchema(json) {
-  const now = nowMoment();
-  const thisYear = Number(now.year());
-
-  Object.keys(json).forEach((field) => {
-    if (field.includes('reportingPeriod')) {
-      json[field]['ui:options'].yearsRange = [thisYear - 2, thisYear + 10];
-    } else if (
-      field.includes('application') ||
-      field.includes('swrsDeadline')
-    ) {
-      json[field]['ui:options'].yearsRange = [thisYear, thisYear + 10];
-    }
-  });
-
-  return json;
-}
```

Comment:
This transformation the old component's uiSchema to limit the range of selectable years (so it didn't show ~100 yrs), which is no longer necessary with the new calendar widget.